Documents/API.md

```diff
@@ -275,7 +277,8 @@ All endpoints in the `/institutions` namespace require two headers (see "Authori
         {
             "edits": [
                 {
-                   "edit": "q007",
+                   "edit": "Q007",
+                   "description: "Description of Q007",
```

Comment:
This `"description` and two others below are missing closing quotes, which is why the highlighting here is weird.